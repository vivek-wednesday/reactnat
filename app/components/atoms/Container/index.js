/**
 *
 * Container
 *
 */

import React from 'react';
import styled from 'styled-components/native';

const StyledContainer = styled.View`
  display: flex;
  flex: 1;
`;
const Container = ({ ...props }) => (
  <StyledContainer testID="container" {...props} />
);

Container.propTypes = {};

export default Container;
