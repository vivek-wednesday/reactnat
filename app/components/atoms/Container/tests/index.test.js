/**
 *
 * Tests for Container
 *
 */
import React from 'react';
import { renderWithIntl } from 'app/utils/testUtils';
import Container from '../index';

describe('<Container />', () => {
  it('should render and match the snapshot', () => {
    const baseElement = renderWithIntl(<Container />);
    expect(baseElement).toMatchSnapshot();
  });

  it('should contain 1 container', () => {
    const { getAllByTestId } = renderWithIntl(<Container />);
    expect(getAllByTestId('container').length).toBe(1);
  });
});
