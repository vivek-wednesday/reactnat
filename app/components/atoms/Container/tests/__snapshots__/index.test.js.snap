// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> should render and match the snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "display": "flex",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
      },
    ]
  }
  testID="container"
/>
`;
